packages/website/versioned_docs/version-latest/introduction/configuration/configuration.md

```diff
@@ -70,7 +70,7 @@ emit:
   - emitter1
 ```
 
-in `<my-pkg>/tspconfig.yaml`, enable `emitter2` using the options specified in the parent `tspconfig.yaml
+in `<my-pkg>/proj2/tspconfig.yaml`, enable `emitter2` using the options specified in the parent `tspconfig.yaml
```

Comment:
In my opinion the second example is not necessary and has been covered above.

Reply:
that's fair, feel free to remove that one

Reply:
removed